### DISCUSSION TITLE
Cookies and url redirections

### DISCUSSION THREAD

**Discussion by adri1g** [usebruno/bruno#1344] [General]
Hello, related to the cookies management topic I have an issue with cookies and url redirections. Some cookies are lost during redirection. Is there a roadmap to cover all these use cases ? It seems maybe linked to axios. Does anyone have the same issue and a potential workaround ?
Anyways thanks for the work done. Bruno is a great tool :)

**Comment by helloanoop** [comment]
@adri1g We will handle the use case around cookie redirection soon.

A temporary workaround is to disable the redirect so that the cookie can be set in Bruno.
You can add `req.setMaxRedirects(0);` to you pre-request script

![image](https://github.com/usebruno/bruno/assets/9350733/554ca0b5-d706-44b1-b1a2-ca93c9bb9a17)

**Reply by adri1g** [reply]
Thanks I used req.setMaxRedirects(0); API and I'm now able to follow the redirection url in a post response script using axios. The pain point was to manage cookies manually for redirection and also header parameters.
Now I have a diplay issue. Inside a script I'm only able to log web service response in the browser console so It's not user friendly. I cannot deploy Bruno like this instead of Postman for my users. I know you are working hard :) I will keep a look on the next releases.
